### PR TITLE
URL sanitizer

### PR DESCRIPTION
**Pull request by babenek**
## Description

Please include a summary of the change and which is fixed.

- Fix URL sanitizer for %XX and \uXX pattern in URL strings
- BM scores update after new obfuscation approach https://github.com/Samsung/CredData/blob/ddc4dedfd2a203e7b502a79ce8b986f16deca65a/download_data.py#L322

## How has this been tested?

Please describe the tests that you ran to verify your changes.

- [x] UnitTest
- [x] Benchmark

### CODE REVIEW

File: .github/workflows/benchmark.yml [hunk 1]
```diff
@@ -22,7 +22,8 @@ jobs:
       - name: Checkout CredData
         uses: actions/checkout@v4
         with:
-          repository: Samsung/CredData
+          repository: babenek/CredData
+          ref: CredData
```
Comment (babenek) [inline .github/workflows/benchmark.yml:26]:
```suggestion
          repository: Samsung/CredData
```

File: .github/workflows/benchmark.yml [hunk 2]
```diff
@@ -72,7 +73,8 @@ jobs:
...
```
Comment (babenek) [inline .github/workflows/benchmark.yml:77]:
```suggestion
          repository: Samsung/CredData
```

File: .github/workflows/benchmark.yml [hunk 3]
```diff
@@ -174,7 +176,8 @@ jobs:
...
```
Comment (babenek) [inline .github/workflows/benchmark.yml:180]:
```suggestion
          repository: Samsung/CredData
```

File: .github/workflows/benchmark.yml [hunk 4]
```diff
@@ -355,7 +358,8 @@ jobs:
...
```
Comment (babenek) [inline .github/workflows/benchmark.yml:362]:
```suggestion
          repository: Samsung/CredData
```

**Review by babenek — COMMENTED** [review]
TODO: rollback after https://github.com/Samsung/CredData/pull/144

**Review by kmnls — COMMENTED** [review]
Agree

**Review by kmnls — APPROVED** [review]
Agree

**Review by Yullia — APPROVED** [review]
Great job!

**Pull request merged**